src/components/App/index.js

```diff
@@ -103,19 +103,23 @@ class App extends React.Component {
       .done(() => this.setState({ sectors: sectorsCollection.toJSON() }));
     regionsCollection.fetch()
       .done(() => this.setState({ regions: regionsCollection.toJSON() }));
+
+    this._updateRouterParams();
```

Comment:
Can you put this line back where it was?
I know that you did it to avoid a reload due to a new setState, but we need this where it was to be sure that all elements all instantiated and receive their properties. 
Clem is working refactoring all this file so for the moment, let's keep it at its original place.